Protocol.md

```diff
@@ -279,6 +279,53 @@ The recipient:
 
   These script pub key forms include only standard forms accepted by the wider set of deployed Bitcoin clients in the network, which increase the chances of successful propagation to miners.
 
+## Optional features
+
+Optional features consist of "features" which parties can opt-in to using. These messages are not needed to facilitate construction of a DLCs.
+
+### The `close_dlc` Message
+
+This message contains information about mutual close, which allows the counterparty
+to broadcast a mutual closing transaction.
+
+1. type: ? (`close_dlc_v0`)
+2. data:
+   * [`32*byte`:`contract_id`]
+   * [`signature`:`close_signature`]
+   * [`u64`:`offerPayoutSatoshis`]
+   * [`u64`:`acceptPayoutSatoshis`]
+   * [`u64`:`fund_input_serial_id`]
+   * [`u16`:`num_funding_inputs`]
+   * [`num_funding_inputs*funding_input`:`funding_inputs`]
+   * [`funding_signatures`:`funding_signatures`]
+
+
+`payout_spk` and `payout_serial_id` from `offer_dlc` as well as `payout_spk` and `payout_serial_id` from `accept_dlc` should be used for constructing the close transaction
+
+`fund_input_serial_id` is a randomly chosen number which uniquely identifies the funding output to be spent.
```

Comment:
calling this `fund_input_serial_id` is a bit confusing when we call the other inputs `funding_inputs` however I can't think of a better name

Reply:
in [Transactions.md](https://github.com/discreetlogcontracts/dlcspecs/blob/master/Transactions.md#funding-output) we call it **Funding Output** but not sure if `funding_output_serial_id` would be appropriate, since it's an input in this case

Reply:
yeah :/ I'm not sure what a better name is

Reply:
I think what should be changed is the `funding_inputs` naming. It conflict with the inputs of the funding transactions and is IMHO quite confusing. What about something like `extra_inputs` or `cancellable_inputs`?